### DISCUSSION TITLE
Disable animations for vertical tab panel?

### DISCUSSION THREAD

**Discussion by dxrcy** [Floorp-Projects/Floorp#1480] [Feature Request]
Apologies if this has been asked before; I couldn't find an existing issue.

I would like to disable animations for the hiding/showing of the vertical tab panel, as it is distracting. Alternatively, I would like to change the animation speed to be much faster.

Thanks!

**Comment by psychananaz** [comment]
bump!!

**Comment by psychananaz** [comment]
> Apologies if this has been asked before; I couldn't find an existing issue.
> 
> I would like to disable animations for the hiding/showing of the vertical tab panel, as it is distracting. Alternatively, I would like to change the animation speed to be much faster.
> 
> Thanks!

I also plays in very low quality, since it is scaled up to fit.

**Comment by CutterKnife** [comment]
The duration of the animation when hovering over the sidebar can be changed using `sidebar.animation.expand-on-hover.duration-ms`.
